app/src/main/res/layout/tab_list_row.xml

```diff
@@ -50,7 +52,11 @@
 
         <ImageView android:layout_width="0dp"
                    android:layout_height="100dp"
-                   android:src="@color/photonBlue40"
+                   android:layout_marginLeft="1dp"
```

Comment:
Always use start/end when they're available over left/right. Left/right are essentially deprecated. Doing otherwise breaks built-in support for bidirectional languages like Hebrew, Arabic, etc. This will matter down the line.